libqtile/layout/slice.py

```diff
@@ -50,6 +50,18 @@ def remove(self, window):
         assert self.window is window
         self.window = None
 
+    def configure(self, window, screen_rect):
+        if window is self._get_window():
+            window.place(
+                screen_rect.x, screen_rect.y,
+                screen_rect.width, screen_rect.height,
+                0,
+                None,
+            )
+            window.unhide()
+        else:
+            window.hide()
+
     def _get_window(self):
```

Comment:
While you're at it...do we really need this method?

Reply:
Nop. :)